storage/innobase/row/row0import.cc

```diff
@@ -1213,6 +1215,21 @@ row_import::match_index_columns(
 
 			err = DB_ERROR;
 		}
+
+		if (cfg_field->descending != field->descending) {
+			ib_errf(thd, IB_LOG_LEVEL_ERROR,
+				ER_TABLE_SCHEMA_MISMATCH,
+				"Index %s field %s is %s which does "
+				"not match metdata file which is %s",
```

Comment:
I think that it suffices to display the ascending/descending only once. For clarify, it may be better to say .cfg file.